packages/lesswrong/server/example.ts

```diff
@@ -0,0 +1,12 @@
+
+export function print123() {
```

Comment:
eh?

Reply:
Oops. Was for testing the REPL, accidentally added during a merge. Removing.